tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -475,6 +475,24 @@ def wait_for_replay_complete(self, imagespec):
                 if int(out.split("=")[-1]) == 0:
                     return 0
 
+    def wait_for_snapshot_complete(self, imagespec):
+        """verify for snapshot completion
+        Args:
+            imagespec: Image specification.
+        """
+        while True:
+            log.info(f"verifying for snapshot completion for {imagespec}")
+            data = self.mirror_status("image", imagespec, "description").split(", ")[1:]
+            for value in data:
+                out = eval(value)
+                if out["local_snapshot_timestamp"] == out["remote_snapshot_timestamp"]:
+                    log.info("Snapshot is successfully completed")
+                return 0
+            else:
```

Comment:
Did you test this change? if and else are at different indentation levels. It should give some error I believe

Reply:
LGTM